state/upgrades.go

```diff
@@ -235,7 +235,7 @@ func DropOldLogIndex(st *State) error {
 	// If the log collection still has the old e,t index, remove it.
 	key := []string{"e", "t"}
 	db := st.MongoSession().DB(logsDB)
-	collection := db.C(logsC)
+	collection := db.C("logs")
```

Comment:
The "logs" collection won't necessarily be there will it? Once we upgrade to 2.2 it will be gone. So this DropOldLogIndex step can be deleted.